tests/unit/source_code/test_pyspark.py

```diff
@@ -17,7 +17,7 @@ def test_spark_sql_no_match(empty_index):
     sqf = SparkSql(ftf, empty_index)
 
     old_code = """
-spark.read.csv("s3://bucket/path")
+spark.read.csv("/bucket/path")
```

Comment:
don't change these. `spark.read.csv("/bucket/path")` actually means `spark.read.csv("dbfs:/bucket/path")`

Reply:
I see. I did not want to pollute those tests with new deprecations from the new matcher was my thinking there, but I can put them back in and change the expected results to include the new deprecations

Reply:
s3: references restored. Additional Advices added to expected Advices, with the exception of tests testing for zero advice returns, where this new matcher is not actually relevant to the test.